app/indexers/describable_indexer.rb

```diff
@@ -109,17 +107,39 @@ def archived_website?
     genres.include?('archived website')
   end
 
+  # TODO: part of https://github.com/sul-dlss/dor_indexing_app/issues/567
+  # From Arcadia, it should be:
+  #  typeOfResource is "text" and any of: issuance is "continuing", issuance is "serial", frequency has a value
+  def periodical?
+    event_selector('publication')&.note&.any? { |note| note.type == 'issuance' && note.value == 'serial' }
+  end
+
+  def pub_date
+    pub_date = PubDateBuilder.build(event_selector('publication'), 'publication')
+    return pub_date if pub_date.present?
+
+    pub_date = PubDateBuilder.build(event_selector('creation'), 'creation')
+    return pub_date if pub_date.present?
```

Comment:
is this check necessary?   can this method be 

```ruby
PubDateBuilder.build(event_selector('publication'), 'publication') ||
  PubDateBuilder.build(event_selector('creation'), 'creation') ||
```

Is pubDateBuilder returning empty strings?

Reply:
feel free to change it - it's a production bug, and I'm out for 2nd jab ... and whatever reaction I get to it.

Reply:
took your excellent suggestion